README.md

```diff
@@ -1,28 +1,31 @@
 
 While `ghc-lib` provides the full GHC API, it doesn't contain a runtime system, nor does it create a package database. That means you can't run code produced by `ghc-lib` (no runtime), and compiling off-the-shelf code is very hard (no package database containing the `base` library). What you can do:
 
-* Parsing Haskell code, as a potential replacement for [`haskell-src-exts`](https://hackage.haskell.org/package/haskell-src-exts). See the demo [`mini-hlint`](https://github.com/digital-asset/ghc-lib/blob/master/examples/mini-hlint/src/Main.hs) in this repo.
-* Compiling Haskell code as far as Core, which includes renaming and type checking. See the demo [`mini-compile`](https://github.com/digital-asset/ghc-lib/blob/master/examples/mini-compile/src/Main.hs) in this repo, and the carefully tailored [file it compiles](https://github.com/digital-asset/ghc-lib/blob/master/examples/mini-compile/test/MiniCompileTest.hs).
+* Parse Haskell code, making `ghc-lib` a potential replacement for [`haskell-src-exts`](https://hackage.haskell.org/package/haskell-src-exts). See the demo [`mini-hlint`](https://github.com/digital-asset/ghc-lib/blob/master/examples/mini-hlint/src/Main.hs) in this repo;
+* Compile Haskell code as far as Haskell's [Core language](https://ghc.haskell.org/trac/ghc/wiki/Commentary/Compiler/CoreSynType), which includes renaming and type checking. See the demo [`mini-compile`](https://github.com/digital-asset/ghc-lib/blob/master/examples/mini-compile/src/Main.hs) in this repo, and the carefully tailored [file it compiles](https://github.com/digital-asset/ghc-lib/blob/master/examples/mini-compile/test/MiniCompileTest.hs).
 
 # Using `ghc-lib`
 
-The package `ghc-lib` will be released on [Hackage](https://hackage.haskell.org/), and can be used like any normal package, e.g. `cabal install ghc-lib`. Since it conflicts perfectly with the GHC API, you may wish to use `PackageImports`, e.g. `import "ghc-lib" GHC`.
+You are free to use `ghc-lib` under the terms of the BSD-3-Clause OR Apache-2.0 licence. See the [LICENSE](https://github.com/digital-asset/ghc-lib/blob/master/ghc-lib-gen/LICENSE) file for details.
+
+The package `ghc-lib` will be released on [Hackage](https://hackage.haskell.org/), and can be used like any normal package, e.g. `cabal install ghc-lib`. Since it conflicts perfectly with the GHC API, you may wish to enable the language extension `PackageImports` so that you can for example, `import "ghc-lib" GHC`.
 
 # Creating `ghc-lib`
 
-To build `ghc-lib` you'll need clones of this repository and the [GHC repository](https://git.haskell.org/ghc.git)(with the git option `--recursive`). Then,
-```
-  cd <path-to-ghc-lib>/ghc-lib-gen
-  cabal run <path-to-ghc>
-  cd <path-to-ghc>
+To build `ghc-lib` you'll need clones of this repository and the [GHC repository](https://git.haskell.org). In a bash shell, build with the following commands.
+```bash
```

Comment:
I would suggest newline before the ''', and not indenting inside the '''

Reply:
✔️